shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -554,7 +555,7 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(dstFile.getAbsolutePath() + randomSuffix);
+    File tmpDst = new File(Name.HOME + randomSuffix);
```

Comment:
If the user runs `cp file /home/username`, could we make the temporary file `/home/username/file.tmp`, then rename it to `/home/username/file` at the end? That's how the current implementation does things for absolute paths.